### PR TITLE
update snap yaml with last release info

### PR DESCRIPTION
**Pull request by leecow**

### CODE REVIEW

**Comment by dagood** [comment]
Looks like the copy/replace was a little off:

src/installer/pkg/packaging/**snaps/snaps**/dotnet-runtime-2.1/snap/snapcraft.yaml

File: src/installer/pkg/packaging/snaps/snaps/dotnet-runtime-2.1/snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: dotnet-runtime-21
+base: core18
+version: 2.1.14
```
Comment (jkotas) [inline src/installer/pkg/packaging/snaps/snaps/dotnet-runtime-2.1/snap/snapcraft.yaml]: Why do we maintain these yaml files for old releases in master?

**Review by Juspem1980 — APPROVED** [review]

**Review by Juspem1980 — APPROVED** [review]

File: src/installer/pkg/packaging/snaps/snaps/dotnet-runtime-2.1/snap/snapcraft.yaml (hunk 1 above)
Comment (deadpaukevich) [reply src/installer/pkg/packaging/snaps/snaps/dotnet-runtime-2.1/snap/snapcraft.yaml]: Совсем недавно открыл для себя гидхаб

**Review by Juspem1980 — APPROVED** [review]

**Comment by stephentoub** [comment]
@leecow, what's the status of this PR?

**Comment by leecow** [comment]
@stephentoub, probably easiest to close this PR and I can get everything up to date with another PR.

**Comment by stephentoub** [comment]
Ok, thanks.

**Pull request closed without merging**